pkg/plugin/admissioncontrollers/webhook/admission_controller.go

```diff
@@ -69,9 +72,27 @@ type ValidateConfResponse struct {
 	Reason  string `json:"reason"`
 }
 
-func (c *admissionController) mutate(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
-	req := ar.Request
-	namespace := ar.Request.Namespace
+func admissionResponseBuilder(uid string, allowed bool, resultMessage string, patch []byte) *v1beta1.AdmissionResponse {
+	res := &v1beta1.AdmissionResponse{}
+	res.Allowed = allowed
+	res.UID = types.UID(uid)
+
+	if len(resultMessage) != 0 {
+		res.Result = &metav1.Status{
+			Message: resultMessage,
+		}
+	}
+	if len(patch) != 0 {
+		res.Patch = patch
+		res.PatchType = func() *v1beta1.PatchType {
+			pt := v1beta1.PatchTypeJSONPatch
+			return &pt
+		}()
```

Comment:
simpler to just do this with the same result and less overhead:

	pt := v1beta1.PatchTypeJSONPatch
	res.PatchType = &pt